src/strategies/rnbw-balance/index.ts

```diff
@@ -0,0 +1,76 @@
+import { formatUnits } from '@ethersproject/units';
+import Multicaller from '../../utils/multicaller';
+
+export const author = 'amaterasu';
```

Comment:
this should be your github username

Reply:
Thank you for reviewing @ChaituVR. The issues are resolved in the latest pushed commit.